### ISSUE TITLE
join efforts in developing the angel framework

### ISSUE THREAD

**Issue by insinfo** [conduit-dart/conduit#215]
@codekeyz @iamEtornam @rasheedmhd @dukefirehawk @j4qfrost @neterror @crifurch  @luis901101
Although I admire your dedication to creating another framework, I think that instead of creating another framework we could join efforts in developing the angel framework here
https://github.com/dukefirehawk/angel

**Comment by j4qfrost** [comment]
Maintenance has to continue on the project. There are still companies who still use this. Honestly, maintenance work on conduit isn't very difficult. It's backwards compatibility requests that are kind of complicated, and every time I sit down to think about how to best make migration tools I get distracted. Part of this is lack of interest on my part, another part is that dart generally pushes forward without too much consideration for migration aside from nnbd, and another is that there's very little outside support, I'm the only one really working on this.

I've felt kind of burnt out on conduit, so I've been working on other projects, but stuff like upgrading the postgres driver is pretty easy because my predecessors set up very good tests, and I set up infrastructure to ensure that the testing remained solid. I still have a lot of cool ideas I want to implement using conduit, but I have a long list of backlog ideas I am working on solo.

TL;DR good luck with angel, I won't be helping out.
